marti_nav_msgs/msg/Obstacle.msg

```diff
@@ -0,0 +1,10 @@
+string id
+# Unique id for the obstacle or empty if not used.
+
+geometry_msgs/PoseStamped pose
+# The pose of the origin of the obstacle.
+
+geometry_msgs/Point[] polygon
```

Comment:
There's a [geometry_msgs/Polygon](http://docs.ros.org/api/geometry_msgs/html/msg/PolygonStamped.html) message, but that might just add an unnecessary layer of indirection.